docs/ca/dev/preparation-for-going-live.md

```diff
@@ -58,76 +54,131 @@ Double check that you do not have any clear text passwords stored in config file
 
 ### Testing
 
 - *Conduct load tests*. Conduct load tests for your application. The sample data used for testing should be comparable to the size and complexity of the production data.
 - *Performance testing and environment scale dial-in*. Testing your production environment before officially going live and assessing its performance are critical steps for a successful launch. Because production environments typically employ horizontal auto-scaling, it's essential to conduct stress and performance tests under expected average and peak loads. These tests enable our team to optimize the environment's vertical scaling in advance, ensuring that it can seamlessly handle the expected loads from the get-go, without any potential delays caused by auto-scaling mechanisms. This proactive approach eliminates the need for post-launch adjustments, providing your team with a significant advantage and peace of mind, while delivering a fast and responsive experience to your users right from the first request to the application.
 To make this process work effectively, maintain active communication with us. Inform us about your load and performance test plans and share the results so that we can fine-tune the environment to meet your specific requirements.
-- Import real data on production.
+- TO BE DISCUSSED *Perform security audits to identify and address vulnerabilities.* 
+- *Import real data on production*. After import is done, it is crucial double-checking the completeness and accuracy of migrated data, especially if transitioning from another platform.
 
 {% info_block warningBox "Data import" %}
 
 You must start working with data of realistic size and quality as early as possible. At this point, you must be importing data regularly to all your environments. We recommend working with the same import data across all your environments. This significantly simplifies troubleshooting and helps you estimate import performance, leading up to your go-live and helping us with environment sizing considerations. We expect all our customers to follow this advice.
 
 {% endinfo_block %}
 
-- Test payment.
+- *Validate Checkout/OMS process*
+  - Test general checkout steps.
+  - Test OMS.
+  - Test shipment.
+  - Test payment.
 
 {% info_block warningBox "Integration with payment providers" %}
 
 Lower or nonproduction environments may not have the same WAF and firewall settings configured as production environments. Therefore, make sure that all your requests have valid headers. Also, test the functionality of payment integrations that use call-backs or depend on API calls to your application in your production environment.
 
 {% endinfo_block %}
 
+### SEO ### 
+
+- *Sitemap is generated*.
+- *Robots.txt is prepared*.
+- *All the content is prepared*. All the CMS pages and other content are prepared (including meta tags, internal and external links, etc).
+
+### Training ### 
+
+- *Make sure that training for staff is done.*:
```

Comment:
This part was already covered previously,  in 8-weeks before golive. Better to remove? If not - please specify what is the focus at this stage

Reply:
Removed